processor/groupbytraceprocessor/event.go

```diff
@@ -162,9 +162,14 @@ func (em *eventMachine) handleEvent(e event) {
 			return
 		}
 
-		em.handleEventWithObservability("onTraceReleased", func() error {
-			return em.onTraceReleased(payload)
-		})
+		// onTraceReleased usually calls next consumers - shouldn't block here.
+		// moreover, common observability doesn't need to be applied to releasing,
+		// because last phase does not affect this processor anymore.
+		go func() {
+			if err := em.onTraceReleased(payload); err != nil {
+				em.logger.Error("onTraceReleased failed", zap.Error(err))
```

Comment:
You'll need to include payload information to make this useful.

Reply:
I don't like the idea to log payload for every error. Just imagine that the following processor starts to fail on every trace. Btw the payload is `ResourceSpans`, what profit do we get from logging pointers? 

Ideally, this logger should be aggregate like in exporters to avoid spamming on failures

Reply:
Adding the traceID would be more than enough. You can also move it to Debug instead. 

> Ideally, this logger should be aggregate like in exporters to avoid spamming on failures

That's the main reason to use metrics for this, so that people know that something went wrong but still don't get flooded with error messages.

Reply:
People know, that something went wrong, but don't know what exactly. That's why every processor returns error - in my opinion it must be shown in logs. This is error indeed. 

If it is needed, why don't you put it into the error? In my world error should be self-sustained

Reply:
> If it is needed, why don't you put traceID into the error? In my world error should be self-sustained

That's what I'm requesting :-)

Reply:
I mean put it in the place where the error occured. Honestly, I don't see any profit in knowing traceID. At least in my cases with sampling I cannot do anything with ID

Reply:
And currently no traceID is logged for error. Don't think recovering traceID from `[]ResourceSpans` is the right move